zabbix.go

```diff
@@ -100,7 +101,7 @@ func NewZabbix(
 
 func (zabbix *Zabbix) restoreSession(path string) error {
 	file, err := os.OpenFile(
-		path, os.O_CREATE|os.O_RDWR, 0600,
+		path, os.O_RDONLY, 0600,
```

Comment:
Wow, what is the point? With `open(O_RDONLY)` file will be not created if does not exists.